docs/src/reference/expressions.md

```diff
@@ -238,6 +251,19 @@ cycle(duration("P7D"))
 // "R/P7D"
 ```
 
+The current date and date-time can be accessed using the built-in functions `today()` and `now()`. In order to store the current date or date-time in a variable, it must be converted to a string using the built-in function `string()`.
```

Comment:
> In order to store the current date or date-time in a variable, it must be converted to a string using the built-in function `string()`

Does that also hold for output mappings?

For example:
source: `= today()`
target: `today`

Reply:
What do you mean exactly? :sweat_smile: 

The conversion to a string is necessary. Otherwise, it would store it as `null`.

Reply:
> Otherwise, it would store it as null.

I think we should add that

Reply:
I'm not sure. If we see that users want to store a date-time value in a variable then we can do the transformation automatically. I think we can wait for the user input :grin:

Reply:
I think the current behaviour does not make sense from a user's perspective. I think one extra line of documentation helps more than waiting for users to tell us something seems wrong.

Reply:
As discussed, let's not bind ourselves on this behaviour by documenting it.